controllers/openstackmachine_controller.go

```diff
@@ -368,6 +407,7 @@ func (r *OpenStackMachineReconciler) reconcileNormal(ctx context.Context, scope
 		err = r.reconcileLoadBalancerMember(scope, openStackCluster, machine, openStackMachine, instanceNS, clusterName)
 		if err != nil {
 			handleUpdateMachineError(scope.Logger, openStackMachine, errors.Errorf("LoadBalancerMember cannot be reconciled: %v", err))
+			conditions.MarkFalse(openStackMachine, infrav1.APIServerIngressReadyCondition, infrav1.LoadBalancerMemberErrorReason, clusterv1.ConditionSeverityError, "Reconciling LoadBalancerMember failed: %v", err)
```

Comment:
```suggestion
			conditions.MarkFalse(openStackMachine, infrav1.APIServerIngressReadyCondition, infrav1.LoadBalancerMemberErrorReason, clusterv1.ConditionSeverityError, "Reconciling load balancer member failed: %v", err)
```

I think it's better to write this out in the condition. The condition is for the end user as well, not only for us devs :)